### PR TITLE
fix symfony 6 warning and cleanup ci

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /tests export-ignore
 /.gitattributes export-ignore
+/.github export-ignore
 /.gitignore export-ignore
-/.php_cs.dist export-ignore
-/.styleci.yml export-ignore
-/.travis.yml export-ignore
```
Comment (dbu) [inline .gitattributes:6]: obsolete files

File: .gitattributes [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+/.php-cs-fixer.dist.php export-ignore
+/phpstan.neon export-ignore
 /phpunit.xml.dist export-ignore
 /ruleset.xml export-ignore
```
Comment (dbu) [inline .gitattributes]: @ChristianRiesen do you know what ruleset.xml is? looks like an obsolete codestyle configuration, do we still need the file in the repository?

File: composer.json [hunk 3]
```diff
@@ -39,10 +39,5 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.0",
         "symfony/phpunit-bridge": "^5.2 || ^6.0"
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.3.x-dev"
```
Comment (dbu) [inline composer.json:45]: using a 1.x branch gets rid of this hack.

File: .gitattributes (hunk 2 above)
Comment (thePanz) [reply .gitattributes]: We would remove it, as we moved to php-cs-fixer.

Wasn't ruleset.xml used for PHP_CodeSniffer maybe?

Comment (dbu) [reply .gitattributes]: i think so. removed the file.

**Pull request merged**